tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -172,23 +167,21 @@ def test_universal_autolog_calls_pyspark_immediately():
 
 
 @pytest.mark.parametrize("config", [{"disable": False}, {"disable": True}])
-def test_universal_autolog_attaches_pyspark_import_hook_if_pyspark_isnt_installed(config):
+def test_universal_autolog_attaches_pyspark_import_hook_in_oss(config):
```

Comment:
Is `not in databricks` equivalent to `in OSS`?

Reply:
I'd rename this test to `test_universal_autolog_attaches_pyspark_import_hook_if_not_in_databricks` to be more accurate.